internal/guibuilder/main.go

```diff
@@ -82,20 +82,19 @@ func (b *Builder) Run() {
 	w, err := storage.Writer(goURI)
 	err = gui.ExportGoPreview(b.root, b.meta, w)
 
+	exec_cmd := func(command string) *exec.Cmd {
+		command_fields := strings.Fields(command)
```

Comment:
Won't this break parameters with spaces? Using the string slice is much safer and not ambiguous.